### PR TITLE
Add nano and vim to Dockerfiles

### PR DESCRIPTION
**Pull request by robinovitch61**
#514 

Add nano and vim editors to Dockerfiles.

### CODE REVIEW

**Comment by Bidek56** [comment]
Nano is probably OK, since the install size is small, vim and dependencies take up a lot of space.
Travis builds already timeout because of all the libraries that have been added.

File: minimal-notebook/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
     texlive-latex-extra \
     texlive-xetex \
     unzip \
-    vim \
+    nano \
```
Comment (parente) [inline minimal-notebook/Dockerfile]: `nano` only needs to be added to this one Dockerfile. All of the other images here inherit from this image and will have it installed. So it's OK to remove all the other changes besides this one line.

**Comment by parente** [comment]
Hi @robinovitch61. I wanted to make sure you saw the one comment I left about your changes. If you're planning on making the change and just haven't had time yet, no problem. If you don't have time, would you like some help getting this PR across the finish line? One of the maintainers here can push an additional commit to your pull request making the change as long as you've checked the "Grant maintainers access" box in your pull request (on the right side of this page I think).

Just let us know, and thanks again for contributing.

**Comment by parente** [comment]
@robinovitch61 Thanks for seeing this through. The build failure is unrelated to your change, so let's go ahead and merge. 🍰

**Pull request merged**